### ISSUE TITLE
Highlight hovered row of the table

### ISSUE THREAD

**Issue by jonas-jonas** [OpenReqEU/eclipse-plugin-vogella#25]
Currently, it's really hard to follow the row to the right, after clicking on the id, to vote on the bug.

It would be great, to have the clicked row highlighted.

**Comment by vogella** [comment]
@SimonScholz this would be useful. Please have a look

**Comment by SimonScholz** [comment]
![image](https://user-images.githubusercontent.com/7559962/56266374-5409fd80-60ec-11e9-84a1-705c5d015bb3.png)
For now only cells can be highlighted, but I already asked Dirk whether it is also possible for the whole row.

**Comment by jnerlich** [comment]
@SimonScholz any update here?

**Comment by LloydThinks** [comment]
@SimonScholz , do you have an update on this?

**Comment by SimonScholz** [comment]
@LloydThinks currently hover highlighting is not a 100 % fit, but sufficient for now.
